tracker/tracker.go

```diff
@@ -113,8 +113,25 @@ type EventTracker struct {
 //
 // Outputs:
 //   - A new instance of the EventTracker struct.
-func NewEventTracker(config *EventTrackerConfig, store store.EventTrackerStore,
+func NewEventTracker(config *EventTrackerConfig, store eventStore.EventTrackerStore,
 	startBlockFromGenesis uint64) (*EventTracker, error) {
+	if config == nil {
+		return nil, fmt.Errorf("invalid configuration. Failed to init Event Tracker")
+	}
+
+	if config.Logger == nil {
+		return nil, fmt.Errorf("invalid configuration, missing logger. Failed to init Event Tracker")
+	}
+
+	if store == nil {
+		var err error
+
+		store, err = eventStore.NewBoltDBEventTrackerStore("./default.db")
```

Comment:
Move the default name of the store to a global variable, and rename it to eventStore.